sqlx-runner/insert_exec_test.go

```diff
@@ -106,35 +108,78 @@ func TestInsertReal(t *testing.T) {
 		InsertInto("people").
 		Columns("name", "email").
 		Record(&person).
-		Returning("id", "created_at").
+		Returning("id", "email", "created_at").
 		QueryStruct(&person)
-	assert.NoError(t, err)
+	require.NoError(t, err)
 	assert.True(t, person.ID > 0)
 	assert.NotEqual(t, person.CreatedAt, dat.NullTime{})
 
 	// Insert by specifying a record (struct)
-	person2 := Person{Name: "Barack"}
+	person2 := Person{
+		Name: "Barack",
+		NullableMap: hstore.Hstore{
+			Map: map[string]sql.NullString{
+				"key1": {String: "value1", Valid: true},
+			},
+		},
+	}
 	person2.Email.Valid = true
 	person2.Email.String = "obama2@whitehouse.gov"
 	err = s.
-		InsertInto("people").Columns("name", "email").
+		InsertInto("people").
+		Columns("*").
+		Blacklist("id", "created_at", "nullable_at").
 		Record(person2).
-		Returning("id").
+		Returning("*").
 		QueryStruct(&person2)
-	assert.NoError(t, err)
+	require.NoError(t, err)
 	assert.True(t, person2.ID > 0)
 	assert.NotEqual(t, person.ID, person2.ID)
+	assert.Equal(t, sql.NullString{String: "value1", Valid: true}, person2.NullableMap.Map["key1"])
 
-	person3 := Person{Name: "Barack", Nullable: nil}
+	// The below test for handling nullability
+	person3 := Person{
+		Name:        "Barack",
+		Nullable:    nil,
+		NullableMap: hstore.Hstore{},
+		NullableAt:  nil,
+	}
 	err = s.
 		InsertInto("people").
-		Columns("name", "nullable").
+		Columns("name", "nullable", "nullable_map").
```

Comment:
should this have nullable_at to prove that writing a null value defaults it to server time?